Task_HelloWorld/index.html

```diff
@@ -1,11 +1,8 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Hello World Problem</title>
```

Comment:
<B>Hello world</b> should be <b>Hello world</b> to ensure proper HTML syntax. HTML tags should be lowercase.